go/tap/tap.pb.go

```diff
@@ -1037,7 +1037,8 @@ type ObserveRequest_Match_Http_StringMatch_Prefix struct {
 
 func (*ObserveRequest_Match_Http_StringMatch_Exact) isObserveRequest_Match_Http_StringMatch_Match() {}
 
-func (*ObserveRequest_Match_Http_StringMatch_Prefix) isObserveRequest_Match_Http_StringMatch_Match() {}
+func (*ObserveRequest_Match_Http_StringMatch_Prefix) isObserveRequest_Match_Http_StringMatch_Match() {
+}
```

Comment:
This formatting change is getting picked up. It also occurs on `master`.

I think we may only be seeing it now because CI has been using Go version `1.10`. When running `master` locally, I am able to reproduce this formatting change.

Now that CI uses a more current Go version, it also requires this change.